cmd-info.c

```diff
@@ -779,11 +779,6 @@ int command_info(int argc, char *argv[], struct opts *opts)
 	if (ret < 0)
 		pr_warn("cannot open data: %s\n", opts->dirname);
 
-	snprintf(buf, sizeof(buf), "%s/info", opts->dirname);
-
-	if (stat(buf, &statbuf) < 0)
-		return -1;
```

Comment:
No, it should save the timestamp at the time of recording.  It currently uses the info file's mtime as the timestamp but it's not reliable as it can be changed anytime.  I'm thinking about adding an item to uftrace_info_bits (both for timestamp and elapsed time) so that it cannot be affected by a possible file modification.